app/controllers/gardens_controller.rb

```diff
@@ -21,6 +21,9 @@ def show
   end
 
   def index
+    @user = current_user
+    @next_seven_days = next_seven_days
+    @facade = GardenFacade.new(current_user)
```

Comment:
One thought if you feel like refactoring - I think this is basically what we have in the dashboard controller, but we could go ahead and move `@user` and `@next_seven_days` into the facade since we have it! So then in the view we'd call `@facade.user` etc. A bit cleaner, and would move the `next_seven_days` method out of the controller. And only be passing one instance variable from the controller to the view.